.github/workflows/bot_pr_approval.yaml

```diff
@@ -8,4 +8,4 @@ jobs:
     uses: canonical/operator-workflows/.github/workflows/bot_pr_approval.yaml@main
     secrets: inherit
     permissions:
-      packages: write
\ No newline at end of file
+      pull_requests: write
```

Comment:
I think this should be the correct key: https://github.com/canonical/github-runner-operators/actions/runs/19548947432/job/55974989722